d3ploy/demand_driven_deployment_inst.py

```diff
@@ -31,139 +31,141 @@ class DemandDrivenDeploymentInst(Institution):
     """
 
     facility_commod = ts.MapStringString(
-        doc="A map of facilities and each of their corresponding" +
-        " output commodities",
-        tooltip="Map of facilities and output commodities in the " +
-        "institution",
-        alias=['facility_commod', 'facility', 'commod'],
-        uilabel="Facility and Commodities"
+        doc="A map of facilities and each of their corresponding"
```

Comment:
I'm not sure moving the + to the next line will work.
Does this work for you? 


I'm surprised you don't get this error:

```
TypeError: bad operand type for unary +: 'str'
```

Reply:
All the changes in this file are pep8 changes when I used `autopep8`

Reply:
I reverted all pep8 changes